heartbeat/docs/monitors/monitor-browser.asciidoc

```diff
@@ -4,6 +4,8 @@
 TIP: Want to get started with synthetic monitoring?
 See the {observability-guide}/synthetics-quickstart.html[quick start guide].
 
+NOTE: While it is possible to configure browser monitors via the heartbeat YAML config files, we highly recommend using project monitors instead! See the https://www.elastic.co/guide/en/observability/current/synthetic-run-tests.html[Elastic synthetics docs].
```

Comment:
nit: section highlight in the redirected url is not great IMO, can we add the anchor to project monitor section to make it clearer? https://www.elastic.co/guide/en/observability/current/synthetic-run-tests.html#synthetic-monitor-choose-project

Reply:
Great point @emilioalvap I've updated all the links